### ISSUE TITLE
"No such file or directory" when running "haxeget install"

### ISSUE THREAD

**Issue by EliteMasterEric** [l0go/haxeget#9]
```
[eric@eric-manjaro ~]$ haxeget install 4.3.3
Downloading Haxe 4.3.3
🎉 Done Downloading!
  [00:00:03] [██████████████████████████████████████████████████████████████████████████████████████████████████████] 9.89 MiB/9.89 MiB (2.77 MiB/s, 0s)The application panicked (crashed).
Message:  called `Result::unwrap()` on an `Err` value: 
   0: No such file or directory (os error 2)

Location:
   src/cache_directory.rs:45

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
Location: src/install_command.rs:35

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
```

```
[eric@eric-manjaro ~]$ RUST_BACKTRACE=full haxeget install 4.3.3
Downloading Haxe 4.3.3
🎉 Done Downloading!
  [00:00:02] [██████████████████████████████████████████████████████████████████████████████████████████████████████] 9.89 MiB/9.89 MiB (3.73 MiB/s, 0s)The application panicked (crashed).
Message:  called `Result::unwrap()` on an `Err` value: 
   0: No such file or directory (os error 2)

Location:
   src/cache_directory.rs:45

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ BACKTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
   1: __libc_start_main<unknown>
      at <unknown source file>:<unknown line>

Run with COLORBT_SHOW_HIDDEN=1 environment variable to disable frame filtering.
Location: src/install_command.rs:35

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ BACKTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
   1: __libc_start_main<unknown>
      at <unknown source file>:<unknown line>

Run with COLORBT_SHOW_HIDDEN=1 environment variable to disable frame filtering.
[eric@eric-manjaro ~]$ 
```

**Comment by l0go** [comment]
Hi, thanks for your report! Yeah, I have been facing this same issue on macos/linux. Not exactly sure of the reason, I don't have the time rn to look into it.

**Comment by l0go** [comment]
"Fixed" in 21876e60aa59c88f179ea8d4e7f4d2169bf0255f
